### ISSUE TITLE
Multiple Arachne Centrals

### ISSUE THREAD

**Issue by PRijnbeek** [EHDEN/ARACHNE#4]
Suppose there are two ARACHNE Centrals (OHDSI and EHDEN). Can a data node be connected to multiple?

I see only one integration option, how does this work?

<img width="621" alt="image" src="https://user-images.githubusercontent.com/7252105/56850941-77615380-6909-11e9-8c45-ba8f2dc832e2.png">

Or is it possible that an Arachne Central is seen as a Data Node? How would that work?

**Comment by gklebanov** [comment]
Yes, great question Peter. The future vision is to have a network of networks design where one Network (one ARACHNE Central) can be integrated and communicate with another Network (second ARACHNE Central) and route communication like that. This is on a roadmap to be implemented later this year.

**Comment by PRijnbeek** [comment]
@gklebanov and @konstjar please update.
